### DISCUSSION TITLE
想从umi转rsbuild, umi自动处理css modules的方案, rebuild有替代吗?有兄弟了解吗.

### DISCUSSION THREAD

**Discussion by lzw10168** [web-infra-dev/rspack#7413] [Q&A]
![image](https://github.com/user-attachments/assets/d13a3c5b-9f74-496f-a299-ec9d6b4d55f2)
有办法通过插件去解决吗?  看了下umi的源码是添加了@umijs/babel-plugin-auto-css-modules这个插件去处理的. 试了下在这里不太行.

**Comment by hardfist** [comment]
https://mp.weixin.qq.com/s/qkXbVJd0xYs8p-FU-njfsw 可以参考这个，有对应的swc插件

**Reply by hardfist** [reply]
swc-plugin-auto-css-modules

**Comment by lzw10168** [comment]
好的，非常感谢



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月02日 11:57 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [web-infra-dev/rspack] 想从umi转rsbuild, umi自动处理css modules的方案, rebuild有替代吗?有兄弟了解吗. (Discussion #7413) |

https://mp.weixin.qq.com/s/qkXbVJd0xYs8p-FU-njfsw 可以参考这个，有对应的swc插件

**Reply by rookit-ljt** [reply]
想问下。umi想用rebuild的打包，它转rebuild是可行的吧
